src/extension.ts

```diff
@@ -10,8 +10,7 @@ export function activate(context: vscode.ExtensionContext): void {
     const createDataPluginCommand = vscode.commands.registerCommand(
         'nipy.createDataPlugin',
         async () => {
-            // eslint-disable-next-line @typescript-eslint/no-floating-promises
-            commands.createDataPlugin();
```

Comment:
I was able to remove the eslint warning after creating the DataPlugin factory in previous PRs.